README

```diff
@@ -0,0 +1,41 @@
+# Ngrok Agent Docker Image
+
```

Comment:
I think we may also want an `## Introduction` section. I'll give it a go:

### Introduction

This repository contains code related to building and releasing the [ngrok docker image](https://hub.docker.com/r/ngrok/ngrok)

Please visit [ngrok's website](https://ngrok.com/) for more information on what ngrok is.

The [ngrok agent docs page](https://ngrok.com/docs) has more information on how to use the ngrok agent.